runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -736,40 +722,27 @@ public void invalidateClientCache() {
             }
         }
 
-        if (!trimmedAddresses.isEmpty()) {
-            if (!options.isIgnoreTrim()) {
-                throw new TrimmedException(trimmedAddresses);
-            }
-
-            log.warn("read: ignoring trimmed addresses {}", trimmedAddresses);
-        }
+        result.keySet().removeAll(compactedAddressses);
```

Comment:
should we add a trace log to print the compacted addresses?

Reply:
The removed addresses have been tracked inside another function checkLogData.